docs/AppService-dotnetcore.md

```diff
@@ -0,0 +1,58 @@
+# Deploy Helium to Azure App Service
```

Comment:
Should a link be created to the parent Helium readme for the dotnet appservice setup?

Reply:
good catch - done